app/src/main/java/org/mozilla/focus/browser/Urls.kt

```diff
@@ -10,5 +10,6 @@ package org.mozilla.focus.browser
 object Urls {
     const val APP_HOME = "firefox:home"
     const val APP_PREFIX = "firefox:"
+    const val DATA_PREFIX = "data:"
```

Comment:
nit: -> `SCHEME_DATA = "data"`

Might want to group separately from the other things too:
```kotlin
SCHEME_...
SCHEME_...

APP_HOME
...
```